samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
+	 * on antenna matrix board.
+	 */
+	dfe-antenna-num = < 10 >;
```

Comment:
We are not going to use configuration for antenna matrix designed by Nordic here?

Reply:
The content of the file was wrong. Updated to correct one.